docs/tables/oci_core_image_custom.md

```diff
@@ -0,0 +1,32 @@
+# Table: oci_core_image_custom
+
+An image is a template of a virtual hard drive. The image determines the operating system and other software for an instance.
```

Comment:
Description in not proper, it should be about custom images, we have the same description in table `oci_core_image`, please check.